lib/koans/10_enums.ex

```diff
@@ -1,32 +1,38 @@
 defmodule Enums do
   use Koans
 
-  koan "knowing how many elements are in a list is important for book-keeping!" do
-    assert Enum.count([1,2,3,4]) == :__
+  koan "Knowing how many elements are in a list is important for book-keeping" do
+    assert Enum.count([1,2,3]) == :__
   end
 
-  koan "..the same applies for counting elements in a map..." do
-    assert Enum.count(%{ :a => 1, :b => 2, :c => 3, :d => 4}) == :__
+  koan "...the same applies for counting elements in a map..." do
+    assert Enum.count(%{ :a => :foo, :b => :bar}) == :__
   end
 
   koan "..or a keyword list..." do
-    assert Enum.count([a: 1, b: 2, c: 3, d: 4]) == :__
+    assert Enum.count([a: 77, b: 23, c: 12, d: 33, e: 90, f: 113]) == :__
   end
 
+  def less_than_five(n), do: n < 5
   koan "Elements can have a lot in common" do
-    assert Enum.all?([1,2,3], fn element -> element < 5 end) == :__
+    assert Enum.all?([1,2,3], &less_than_five/1) == :__
   end
 
+  def less_than_two(n), do: n < 2
```

Comment:
Same question mark thing.. if we want to follow the styleguide to the letter, this would have a trailing question mark.